File: Selenium/Activities/Activity9.py
# Input Events #2
# Using Selenium:
# Open a new browser to https://training-support.net/webelements/keyboard-events
# Get the title of the page and print it to the console.
# On the page, type out a string from the Selenium script to show on the page
# Print the message to the console.
# Close the browser.
from selenium import webdriver
from selenium.webdriver import ActionChains, Keys
from selenium.webdriver.common.by import By

driver = webdriver.Firefox()
driver.get("https://training-support.net/webelements/keyboard-events")
print("The title of the page is: ", driver.title)
actions = ActionChains(driver)
actions.send_keys("This is the typed text").perform()
typedtext = driver.find_element(By.XPATH, "//h1[contains(@class,'text-center') and contains(@class,'text-4xl')]").text
print("The entered text is: ", typedtext)
driver.quit()

File: Selenium/Activities/Activity9.java
//Input Events #2
//Using Selenium:
//Open a new browser to https://training-support.net/webelements/keyboard-events
//Get the title of the page and print it to the console.
//On the page, type out a string from the Selenium script to show on the page
//Print the message to the console.
//Close the browser.

package activities;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity9 {
	public static void main(String[] args) {
		
		WebDriver driver = new FirefoxDriver();
		driver.get("https://training-support.net/webelements/keyboard-events");
		System.out.println("The title of the page is: " + driver.getTitle());
		
		Actions builder = new Actions(driver);
		builder.sendKeys("This is the typed text").perform();
		
		WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5));
		String typetext = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//h1[contains(@class,'text-center') and contains(@class,'text-4xl')]"))).getText();
		System.out.println(typetext);
		
		driver.quit();
		
	}

}
